tests/lib/bin/session-tool

```diff
@@ -0,0 +1,110 @@
+#!/bin/sh -e
+if [ "$(id -u)" -ne 0 ]; then
+    echo "session-tool needs to be invoked as root" >&2
+    exit 1
+fi
+if [ -z "$(command -v busctl)" ]; then
+    echo "session-tool requires busctl" >&2
+    exit 1
+fi
+if [ $# -eq 0 ]; then
+	echo "usage: session-tool [-u USER] [--] <CMD>"
+	exit 1
+fi
+user=root
+while [ $# -gt 0 ]; do
+	case "$1" in
+		--)
+			shift
+			break
+			;;
+		-u)
+			if [ $# -eq 1 ]; then
+				echo "session-tool: option -u requires an argument" >&2
+				exit 1
+			fi
+			user="$2"
+			shift 2
+			;;
+		-*)
+			echo "session-tool: unsupported argument $1" >&2
+			exit 1
+			;;
+		*)
+			break
+			;;
+	esac
+done
+
+sed -i -e 's/mesg n .*true/tty -s \&\& mesg n/g' /root/.profile
```

Comment:
could you add a comment about why this snippet is necessary ?

Reply:
Sure. I should probably not have left it in here but instead adjusted spread's project wide prepare.

It's to avoid a bug in some systems (notably Debian) that have incorrect /home/root/.profile code which fails in non-interactive shells.

Reply:
Were you going to move this to the project wide prepare?

Reply:
ah I see the TODO now